<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPreTranslationCheck</class>
 <widget class="QDialog" name="DialogPreTranslationCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Webpage loading check</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonPaste">
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUpload">
       <property name="text">
        <string>Upload again</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Copy at least the 10 first lines to check that the webpage were well loaded.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetTrans">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Text to translate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Text loaded</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPreTranslationCheck</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPreTranslationCheck</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
